test/Qwiq.Integration.Tests/WireMock/WireMockRestStoreContext.cs

```diff
@@ -97,6 +140,28 @@ public WireMockRestStoreContext()
             // The WorkItemTrackingClient property is lazily initialized when first accessed.
         }
 
+        /// <summary>
+        /// Determines if an exception indicates an SSL certificate binding failure.
+        /// </summary>
+        private static bool IsSslBindingFailure(Exception ex)
+        {
+            // Check for common SSL binding failure patterns
+            // AggregateException wraps the actual HttpListenerException
+            if (ex is AggregateException aggEx && aggEx.InnerExceptions.Any(IsSslBindingFailure))
+            {
+                return true;
+            }
+
+            // HttpListenerException with error code 5 (Access Denied) or 183 (Cannot create file)
+            // indicates SSL certificate binding issues
```

Comment:
Fixed in commit 3851bac4. Replaced the `foreach` loop with LINQ `Any()` method as suggested:

```csharp
if (ex is AggregateException aggEx && aggEx.InnerExceptions.Any(IsSslBindingFailure))
{
    return true;
}
```